src/lucky_migrator/runner.cr

```diff
@@ -40,6 +40,15 @@ class LuckyMigrator::Runner
     end
   end
 
+  def run_next_migration
+    setup_migration_tracking_tables
+    if pending_migrations.empty?
+      puts "Did nothing. No pending migrations.".colorize(:green)
```

Comment:
Since this has a lot of similarity with `run_pending_migration`, I think it would be good to extract a method. Here's what I'm thinking:

```crystal
def run_next_migration
  prepare_for_migration do
    pending_migrations.first.new.up
  end
end

private def prepare_for_migration
  setup_migration_tracking_tables
  if pending_migrations.empty?
    puts "Did nothing. No pending migrations.".colorize(:green)
  else
    yield
  end
end
```

This will remove most of the duplication between `run_pending_migrations` and `run_next_migration`. What do you think?

Reply:
That's a great idea, I've made the change.